.github/actions/setup-dav1d/action.yml

```diff
@@ -0,0 +1,82 @@
+name: Dav1d
+description: Compile and install dav1d
+
+runs:
+  using: composite
+  steps:
+    # Linux Section
+    - name: Install nasm
+      if: runner.os == 'Linux'
+      uses: ilammy/setup-nasm@v1
+
+    - name: Install Python 3.9
+      if: runner.os == 'Linux'
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.9'
+
+    - name: Install pip packages
+      if: runner.os == 'Linux'
+      run: |
```

Comment:
All of these `run` clauses need a `shell` added:

- `bash` for linux
- `powershell` for windows